api-client/src/camera/types.ts

```diff
@@ -17,6 +17,6 @@ export interface CreateCameraData {
   data: CameraData
 }
 
-export type DownloadedPreviewImageFileResponse = Blob
+export type DownloadedPreviewImageFileResponse = Blob | null
```

Comment:
```suggestion
export type DownloadedPreviewImageFileResponse = Blob
```
Hmmm, I'm pretty sure this should always be defined. The endpoint always returns an image file.